x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.blobcache;
+
+import org.elasticsearch.telemetry.TelemetryProvider;
+import org.elasticsearch.telemetry.metric.LongCounter;
+import org.elasticsearch.telemetry.metric.LongHistogram;
+import org.elasticsearch.telemetry.metric.MeterRegistry;
+
+public class BlobCacheMetrics {
+    private final LongCounter cacheMissCounter;
+    private final LongHistogram cacheMissLoadTimes;
+
+    public BlobCacheMetrics(MeterRegistry meterRegistry) {
+        this(
+            meterRegistry.registerLongCounter(
+                "elasticsearch.blob_cache.miss_that_triggered_read",
```

Comment:
I updated the metric name the prefix already denotes that this is for the blob cache.

Reply:
@stu-elastic also opened a [PR](https://github.com/elastic/elasticsearch-serverless/pull/1106) for this.